.prettierrc

```diff
@@ -6,7 +6,7 @@
   "printWidth": 150,
   "jsxSingleQuote": true,
   "bracketSameLine": false,
-  "endOfLine": "crlf",
+  "endOfLine": "auto",
```

Comment:
döntsük el, hogy mit használunk

Reply:
amúgy konzisiteon is auto van, némelyik projektben nincs is erre szabály. szóval ha ez így nem okoz gondot, akkor maradhat